packages/commonwealth/client/scripts/views/components/Breadcrumbs/Breadcrumbs.tsx

```diff
@@ -0,0 +1,176 @@
+import 'components/Breadcrumbs/Breadcrumbs.scss';
+import React from 'react';
+import { useLocation } from 'react-router-dom';
+import { CWText } from '../component_kit/cw_text';
+import { CWTooltip } from '../component_kit/cw_popover/cw_tooltip';
+import app from 'state';
+
+export const Breadcrumbs = () => {
```

Comment:
this would be good as a "container" holding the logic, but we already have `CWBreadcrumbs` component, which is used btw in some sites, so currently in those sites there are 2 breadcrumbs displayed which is not good. 

![image](https://github.com/hicommonwealth/commonwealth/assets/14819225/327cb04c-08cd-4c0f-b075-e4d5fddbb126)

- you should take CWBreadcrumbs component and adjust it into new styling
- you should remove usage of CWBreadcrumbs from all sites
- you should use CWBreadcrumbs in you Breadcrumbs component to display the new breadcrumbs

Basically `Breadcrumbs` should be container which handles the logic and `CWBreadcrumbs` should be a dumb component that only displays the breadcrumbs passed as a prop.

Reply:
?

Reply:
Resolved this in most recent commit.

Reply:
![Screenshot](https://github.com/hicommonwealth/commonwealth/assets/35875283/b2a8e008-1ac5-4385-aaee-241410b8fdcf)

Putting a screenshot here just to document that this is done incase we have to come back, just for my own sanity.